### ISSUE TITLE
JavaScript Course: Update codeblock delimiters per layout style guide

### ISSUE THREAD

**Issue by aconstas** [TheOdinProject/curriculum#26913]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

Codeblocks should be updated to align with new changes in our [layout style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

In the lessons named in this issue, codeblocks will be opened by a set of 3 tildes (the ~ character) and then closed by another set of 3 tildes. To align with the style guide, these tilde characters need to change to backticks (the ` character). Be sure to refer to [the codeblocks section](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#codeblocks) of the style guide for an example of what this should look like.

### Guidance:

 - Comment on this issue to get a lesson assigned to you. You can also ask for a specific one
 - Please wait until you are assigned by a maintainer before starting to work on this issue to avoid multiple people working on the same lessons
 - Make sure to read the [contributing guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md) before starting your first contribution
 - Please add the following to your PR in the Issues section: Related to #xxxxx, since this issue is divided into sections, individual PRs should **NOT INCLUDE** the closes keyword.

### Lessons to update
- [x] @janaiscoding [ES6 Modules](https://www.theodinproject.com/lessons/node-path-javascript-es6-modules)
- [x] @VMadhuranga [Factory Functions and Module Pattern](https://www.theodinproject.com/lessons/node-path-javascript-factory-functions-and-the-module-pattern)
- [x] @leonardo-vic3nte [Objects and Object Constructors](https://www.theodinproject.com/lessons/node-path-javascript-objects-and-object-constructors)
- [x] @ritizzzz [OOP Principles](https://www.theodinproject.com/lessons/node-path-javascript-oop-principles)
- [x] @mcabrera108 [Project Library](https://www.theodinproject.com/lessons/node-path-javascript-library)
- [x] @riarsdev [Webpack](https://www.theodinproject.com/lessons/node-path-javascript-webpack)

@JoshDevHub 

### Path

Node / JS

### Lesson Url

javascript/organizing_your_javascript_code

**Comment by janaiscoding** [comment]
Hello,
I would be glad if I could participate to this issue 🙏🏼

**Comment by VMadhuranga** [comment]
I would also like to help

**Comment by leonardo-vic3nte** [comment]
I would like to help

**Comment by JoshDevHub** [comment]
Thanks for volunteering everyone!

@janaiscoding I've assigned you the ES6 Modules lesson

@VMadhuranga I've assigned you the Factory Functions and Module Pattern lesson

@leonardo-vic3nte I've assigned you the Object and Object Constructors lesson

**Comment by ritizzzz** [comment]
I would like to help

**Comment by JoshDevHub** [comment]
Awesome @ritizzzz thank you for volunteering.

I've assigned you the OOP Principles lesson

**Comment by mcabrera108** [comment]
I would like to help on this issue!

**Comment by riarsdev** [comment]
Count me in! I'd love to work on this issue.

**Comment by JoshDevHub** [comment]
Thank y'all for volunteering!

@mcabrera108 I've assigned you the Library project

@riarsdev I've assigned you the Webpack lesson

**Comment by JoshDevHub** [comment]
Thank you all for your contributions
